packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -711,15 +710,13 @@ void main() {
     expect(binding.frameScheduled, isFalse);
   });
 
-  test('Service extensions - saveCompilationTrace', () async {
+  test('Service extensions - brightnessOverride', () async {
     Map<String, dynamic> result;
-    result = await binding.testExtension('saveCompilationTrace', <String, String>{});
-    final String trace = String.fromCharCodes((result['value'] as List<dynamic>).cast<int>());
-    expect(trace, contains('dart:core,Object,Object.\n'));
-    expect(trace, contains('package:test_api/test_api.dart,::,test\n'));
-    expect(trace, contains('service_extensions_test.dart,::,main\n'));
-  }, skip: isBrowser); // Compilation trace is Dart VM specific and not
-  // supported in browsers.
+    result = await binding.testExtension('brightnessOverride', <String, String>{});
+    final String brightnessValue = result['value'] as String;
+
+    expect(brightnessValue, 'Brightness.light');
```

Comment:
Can dark be tested?

Reply:
I can't actually change the value in this test or I'll trigger reassemble and break the test framework. I will update the integration test

Reply:
the integration test now confirms both brightness settings and that a bogus value does not change anything